zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -63,37 +63,37 @@
 public class ZeppelinServer extends Application {
   private static final Logger LOG = LoggerFactory.getLogger(ZeppelinServer.class);
 
-  private SchedulerFactory schedulerFactory;
-  public static Notebook notebook;
-  public static NotebookServer notebookServer;
-  public static Server jettyServer;
+  public static Notebook NOTEBOOK;
+  public static NotebookServer NOTEBOOK_SERVER;
```

Comment:
I thinks this should be in camel
according to https://google.github.io/styleguide/javaguide.html#s5.2.5-non-constant-field-names